src/utils/getAttributes.js

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
I hope this comment explains the need for these utility functions. Please let me know if it's unclear, and I can clarify.

Reply:
This is very clear! I like the examples :100:

Reply:
Good point, Dennis. I'll change the name.